app/views/comments/_form.html.erb

```diff
@@ -18,7 +18,7 @@
 
     <div class="mb-3">
       <%= form.label :body, class: "sr-only" %>
-      <%= form.text_area :body, class: "form-control" %>
+      <%= form.text_area :body, class: "form-control", id: "comment_body" %>
```

Comment:
You do not need the  "comment_body" id here, when you inspect the the comment form, you'll see that the id already exists without you creating it so you can remove this!